include/deal.II/fe/fe.h

```diff
@@ -2275,6 +2275,18 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
 
   //@}
 
+  /**
+   * For a finite element of degree @p sub_degree < @p degree, we return a
+   * vector which maps the numbering on an FE of degree @p sub_degree into the
+   * numbering on this element.
+   *
+   * This function is appropriate for hierarchical
+   * elements (such as FE_Q_Hierarchical or FE_Nedelec). Other elements,
+   * therefore, may not implement it.
+   */
+  virtual std::vector<unsigned int>
+  get_embedding_dofs(const unsigned int sub_degree) const;
```

Comment:
Yes, since the one to one identification is only possible in special cases I agree that it makes more sense to leave this out of the base class.

I'll take a look at implementing `get_interpolation_matrix()` for a better generalization here.